lib/manual_and_sections_redirecter.rb

```diff
@@ -0,0 +1,74 @@
+require "gds_api/content_store"
+
+class ManualAndSectionsRedirecter
+  def initialize(args)
+    @publishing_api ||= SpecialistPublisher.services(:publishing_api)
+    @content_store ||= GdsApi::ContentStore.new(Plek.current.find("content-store"))
```

Comment:
Totally minor: we don't need optional assignment (`||=`) on these two lines because you can't set `@publishing_api` or `@content_store` before we call `initialize` so a simple assignment (`=`) will do.  The optional assignment will give someone looking at it pause wondering "is something special happening here" when the answer is "no" so while it's programatically harmless, it's still worth fixing.